lib/chart/internal/axis_line/major_ticks_text.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Chart.Internal.AxisLine.MajorTicksText do
+  @moduledoc false
+
+  alias Chart.Internal.AxisLine.Helpers
+  import Chart.Internal.Guards, only: [is_decimals: 1, is_numbers: 2]
+
+  @self_key :major_ticks_text
+
+  def new() do
+    %{
+      format: {:decimals, 1},
+      gap: 0,
+      labels: [],
+      positions: [],
+      range: {0, 1}
+    }
+  end
+
+  def add(settings, axis) when is_map(settings) and is_atom(axis) do
+    put_in(settings, [axis, @self_key], new())
+  end
+
+  # Setters
+
+  @doc """
+  format :: {:decimals, non_neg_integer()} | {:datetime, String.t()}
+  """
```

Comment:
nemuzes to normalne o typovat?

Reply:
Ja bych to otypoval az to bude vse ok + pridal k tomu vice doc. Zatim bych to ponechal...